### ISSUE TITLE
The element has incomplete content / The element has unexpected child element

### ISSUE THREAD

**Issue by swidz** [salvois/LargeXlsx#50]
After generating an excel file from `System.Data.DataTable` I get the error when opening the file :

_We found a problem with some content in 'abc.xlsx'. Do you want us to try to recover as much as we can? If you trust the source of this workbook, click Yes._

After validating with OOXML validator
[https://marketplace.visualstudio.com/items?itemName=mikeebowen.ooxml-validator-vscode](https://marketplace.visualstudio.com/items?itemName=mikeebowen.ooxml-validator-vscode)

I see the following errors

```
Id : Sch_IncompleteContentExpectingComplex
Description: The element has incomplete content. List of possible elements expected: <http://schemas.openxmlformats.org/spreadsheetml/2006/main:col>.
XPath : /x:worksheet[1]/x:cols[1]
Part URI : /xl/worksheets/sheet1.xml
NamespacesDefinitions undefined

Id: Sch_UnexpectedElementContentExpectingComplex
Description: The element has unexpected child element 'http://schemas.openxmlformats.org/spreadsheetml/2006/main:b'.
XPath: /x:styleSheet[1]/x:fonts[1]/x:font[2]
Part URI: /xl/styles.xml
NamespacesDefinitions: undefined
```
 
Message After repair:

```
Excel completed file level validation and repair. Some parts of this workbook may have been repaired or discarded.
Repaired Part: /xl/worksheets/sheet1.xml part with XML error.  HRESULT 0x8000ffff Line 3, column 0.
```

It seems that adding all columns as unformatted, produce empty <cols></cols> section.
If I add all columns as formatted, the error is gone.

The beging of the file sheet1.xml without formatted columns

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><worksheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main"><sheetViews><sheetView showGridLines="1" showRowColHeaders="1" rightToLeft="0" workbookViewId="0">
</sheetView></sheetViews>
<cols></cols>
```

while with formatted looks like

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><worksheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main"><sheetViews><sheetView showGridLines="1" showRowColHeaders="1" rightToLeft="0" workbookViewId="0">
</sheetView></sheetViews>
<cols><col min="1" max="1" width="30" customWidth="1"/>
<col min="2" max="2" width="30" customWidth="1"/>
<col min="3" max="3" width="30" customWidth="1"/>
<col min="4" max="4" width="30" customWidth="1"/>
</cols>
```

The code:

```
var columns = new XlsxColumn[columnCount];
for (int colIndex = 0; colIndex < columnCount; colIndex++)
{
    //columns[colIndex] = XlsxColumn.Formatted(30); //this works ok
    //columns[colIndex] = XlsxColumn.Unformatted(); //this creates file with errors
}
```

The second issue was created by adding styles, from the example on the LargeXlsx github page.

**Comment by salvois** [comment]
Thanks for the detailed report @swidz ! I hope to have a look in the next days.

**Comment by salvois** [comment]
Fixed in 1.9.2, thanks!
